### ISSUE TITLE
Frontend multi-language support

### ISSUE THREAD

**Issue by mrmueller** [Azure-Samples/azure-search-openai-demo#1690]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>
The frontent is available in the English language only. We translate strings directly in the code to German.
Unfortunately this leaves behind some of our users who speak English, French or Spanish.

### Expected/desired behavior
>
The app supports multiple languages and automatically adapts to the user's browser language.

### Mention any other details that might be useful

- We would be happy to contribute translations, are however not sure how multi-language support could be implemented in the frontend from a code perspective.
- In a multi-language environment like we have it, we can expect the indexed documents to be in multiple languages as well.
The creation of search terms (Backend) does not currently account for that.

**Comment by pamelafox** [comment]
Thanks for filing, I was actually hoping to tackle this soon, so I'm glad to see there's customer interest. I am currently investigating what i18n frameworks to use on the client, I've found ngettext and i18nnext so far. Do you have any that you recommend?

**Comment by mrmueller** [comment]
I like to hear that @pamelafox. Unfortunately, I have zero experience in this topic. Learning about i18n frameworks right now.
Going to add one thing to the feature request:

In a multi-language environment like we have it, we can expect the indexed documents to be in multiple languages as well.
The creation of search terms does not currently account for that.

**Comment by pamelafox** [comment]
Please check out the PR by @bnodir here and see if you have any thoughts on the approach:
https://github.com/Azure-Samples/azure-search-openai-demo/pull/1790

**Comment by mrmueller** [comment]
Hi @pamelafox 

I like the approach for the frontend and will be happy to provide a file with German translations once this is merged.

Retrieval of multi-language content will remain a challenge. I realize that this is harder to solve though. We can create a new issue if it helps.

**Comment by pamelafox** [comment]
Since the PR was merged for this, I'll close this issue. Feel free to open other issues about other multi-lingual issues.
